modules/gpaas-postgres-migrator/extract_task.tf

```diff
@@ -21,7 +21,7 @@ module "extract_task" {
       # N.B. $DUMP_FILENAME is injected by the Step Function task
       override_command = [
         "sh", "-c",
-        "apk update && apk add --no-cache postgresql-client && cf install-plugin -f conduit && cf login -a ${var.cf_config.api_endpoint} -u $CF_USERNAME -p $CF_PASSWORD -o ${var.cf_config.org} -s ${var.cf_config.space} && cf conduit ${var.cf_config.db_service_instance} -- pg_dump --file $DUMP_FILENAME --no-acl --no-owner"
+        "apk update && apk add --no-cache postgresql-client && cf install-plugin -f conduit && rm -rf $DUMP_FILENAME && cf login -a ${var.cf_config.api_endpoint} -u $CF_USERNAME -p $CF_PASSWORD -o ${var.cf_config.org} -s ${var.cf_config.space} && cf conduit ${var.cf_config.db_service_instance} -- pg_dump -j 8 -Fd --file $DUMP_FILENAME --no-acl --no-owner"
```

Comment:
Well now, today I learned that `rm -rf` returns a zero exit code even if the file does not exist, which is cool